test/data/api-methods.js

```diff
@@ -40,7 +36,6 @@ define({
     getPosition: null,
     getProvider: null,
     getQualityLevels: null,
-    getRenderingMode: null,
     getSafeRegion: null,
     getState: null,
```

Comment:
Now that we have a list of deprecated API methods in the test/data, can you add the ones you've remove here to that file?